infra/common/src/main/java/cn/hippo4j/common/extension/design/AbstractSubjectCenter.java

```diff
@@ -73,12 +74,12 @@ public static void register(String subject, Observer observer) {
     }
 
     /**
-     * Remove observer.
-     *
-     * @param observer
+     * remove observe
```

Comment:
Why modify this logic?

Reply:
The remove method is not easy to use

Reply:
This is the default remove method, if you want to add an easier-to-use method, please develop it separately.

Also, pay attention to the code formatting. Please keep the original format.

Reply:
ok